effects/monix/src/main/scala/sttp/client3/impl/monix/MonixWebSockets.scala

```diff
@@ -30,7 +31,10 @@ object MonixWebSockets {
       )
         .mapEval(ws.send(_))
         .completedL
-        .guarantee(ws.close())
+        .guaranteeCase {
```

Comment:
why, isn't close() idempotent?

Reply:
right, I will adjust that in `WebSocketImpl`